### DISCUSSION TITLE
Preload a model into memory at startup

### DISCUSSION THREAD

**Discussion by bnusunny** [mudler/LocalAI#559] [Q&A]
Hello, 

Is it possible to preload one model into memory at startup? The model file is already downloaded and saved in models folder. I want to reload the model into memory when a Lambda function starts up with provisioned concurrency. This will avoid the long wait when people actually invoke the function. 

The current preload parameters seem to download the model from online, not from local storage.

**Comment by Taikono-Himazin** [comment]
Is there any progress on this? I think this feature is necessary.
It is quite frustrating for users to have to wait tens of seconds for the first request.
